kafka-client/src/test/java/dev/responsive/kafka/testutils/IntegrationTestUtils.java

```diff
@@ -336,6 +338,21 @@ public static void startAppAndAwaitRunning(
     }
   }
 
+  public static Merger<String, String> sessionMerger() {
```

Comment:
personal preference, but I prefer to have these in the test file directly so I don't need to read two files (especially when reviewing) to see what a test is doing especially since it's important to the output